docs/source/tutorials_wine.rst

```diff
@@ -0,0 +1,590 @@
+We can download it from storage that is described in ``models-output`` connection (currently this connection
+is created on Legion Platform installation stage, so we have not created this connection above).
+
+But we will just use ``artifactName`` as one of parameters for model packaging
+
+
+Packaging
+~~~~~~~~~~~~~
+
+Let’s check ``packaging.legion.yaml`` file. It contains payload for Legion API call to run model training
+
+.. code-block:: console
+   :linenos:
+   :emphasize-lines: 6,8,10
+
+    $ cat ./legion/training.legion.yaml
+
+    id: wine
+    kind: ModelPackaging
+    spec:
+      artifactName: "<fill-in>"
+      targets:
+        - connectionName: docker-ci
+          name: docker-push
+      integrationName: docker-rest
+
+In this file, we're:
+
+- line 6: set artifact name from previous step;
+- line 8: set docker repository connection where our packaged model will be saved
+- line 10: set Model packaging toolchain integration as rest service
+
+For now we can create model packaging
+
+`Using legionctl`
+
+.. code-block:: console
+
+    $ legionctl packaging create -f ./legion/packaging.legion.yaml
+
+You can see packaging logs using
+
+.. code-block:: console
+
+    $ legionctl packaging logs --id wine
+
+To check status of packaging
+
+.. code-block:: console
+
+    $ legionctl packaging get --id wine
+
+
+`Using jupyterlab`
+
+Submitting the payload, checking logs and results are absolutely the same as for model training process
+
+Result of model packaging is ``image`` artifact that you can find the same way as ``artifactName`` for model training.
+
+This image is stored in docker registry, which we listed when we have created connection at `Pre-sequences` step
+
+
+Deployment
+~~~~~~~~~~~~~
+
+Let’s check ``deployment.legion.yaml`` file. It contains payload for Legion API call to run model training
+
+.. code-block:: console
+   :linenos:
+   :emphasize-lines: 6,8,10
+
+    $ cat ./legion/deployment.legion.yaml
+
+    id: wine
+    kind: ModelDeployment
+    spec:
+      image: "<fill-in>"
+      minReplicas: 1
+      ImagePullConnectionID: docker-ci
+
+In this file, we're:
+
+- line 6: set image name that we got on previous step
+- line 8: set docker repository connection
+
+For now we can create model deployment
+
+`Using legionctl`
+
+.. code-block:: console
+
+    $ legionctl deployment create -f ./legion/deployment.legion.yaml
+
+You can see deployment logs using
+
+.. code-block:: console
+
+    $ legionctl deployment logs --id wine
+
+To check status of deployment
+
+.. code-block:: console
+
+    $ legionctl deployment get --id wine
+
+
+`Using jupyterlab`
+
+Submitting the payload, checking logs and results are absolutely the same as for model training or packaging process.
+
+
+Using
+~~~~~~~~~~~
+After model is successfully deployed you can check its API in swagger.
+
+Just open ``edge.<your-legion-platform-host>/swagger/index.html`` and look and next endpoints
```

Comment:
Add the [example](https://github.com/legion-platform/legion/blob/develop/legion/tests/e2e/robot/tests/e2e/e2e_mlflow_wine.robot#L43) of legionctl command